docs/support_files/api_reference.md

```diff
@@ -60,7 +61,7 @@ Multiple `AfterAll` hooks are executed in the **reverse** order that they are de
 
 #### `Before([options,] fn)`
 
-Defines a hook which is run before each scenario. Same interface as `After` except the first argument passed to `fn` will be an object of the form `{sourceLocation: {line, uri}}` matching the event data for `test-case-started`.
+Defines a hook which is run before each scenario. Same interface as `After` except the first argument passed to `fn` will be an object of the form `{sourceLocation: {line, uri}, pickle: { tags: [{ name, location: { line, column } }], name, language, locations: [{ line, column }], steps: [{ text, arguments: [], locations: [{ line, column }] }]}}` matching the event data for `test-case-started`.
```

Comment:
Thoughts on saying the "will be an object of the form `{sourceLocation, pickle}` (see After hook for more details)" instead of repeating the pickle. Alternatively can reference the other way where they are defined only here and not in the After hook.

Reply:
Agreed I cleaned it up. I put the details in the after hook since that is what comes first in the doc.